ui/apps/platform/src/Containers/Policies/PatternFly/Wizard/Step3/BooleanPolicyLogicSection.tsx

```diff
@@ -24,18 +25,16 @@ function BooleanPolicyLogicSection({ readOnly = false }: BooleanPolicyLogicSecti
     const { values } = useFormikContext<Policy>();
 
     const isImageSigningEnabled = useFeatureFlagEnabled('ROX_VERIFY_IMAGE_SIGNATURE');
+    const isNetworkPolicyFieldsEnabled = useFeatureFlagEnabled('ROX_NETPOL_FIELDS');
     React.useEffect(() => {
         if (values.eventSource === 'AUDIT_LOG_EVENT') {
             setDescriptor(auditLogDescriptor);
         } else {
-            const descriptors = isImageSigningEnabled
-                ? [
-                      ...policyConfigurationDescriptor,
-                      ...networkDetectionDescriptor,
-                      imageSigningCriteriaDescriptor,
-                  ]
-                : [...policyConfigurationDescriptor, ...networkDetectionDescriptor];
-            setDescriptor(descriptors);
+            setDescriptor([
+                ...policyConfigurationDescriptor,
+                ...networkDetectionDescriptor,
+                ...(isImageSigningEnabled ? [imageSigningCriteriaDescriptor] : []),
```

Comment:
Perhaps we can move this logic elsewhere and re-use the code instead of copy-pasting it. When I touched this code recently it already felt like a mild decision since there are 3 places in the code we would need to change instead of one.

Reply:
@pedrottimark sorry for the late response - sounds good to me. When I was touching `descriptors.tsx`, I had a similar idea as well :)